src/sdk/lsf-sdk.js

```diff
@@ -97,7 +98,7 @@ export class LSFWrapper {
 
     if (this.labelStream) {
       interfaces.push("infobar");
-      interfaces.push("topbar:prevnext");
+      interfaces.push("topbar:prev-next-history");
```

Comment:
Although I like the idea of naming this better, I wonder if this will break Opensource usages entirely.

Reply:
it is not being used in lso yet. lso will get all the buttons